resources/definitions/hellbot_magna_SE.def.json

```diff
@@ -0,0 +1,41 @@
+{
+    "version": 2,
+    "name": "Hellbot Magna SE",

```

Comment:
I couldn't find any information on this printer online. Is it being sold anywhere? Is there a sales page about it online?